coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -536,6 +561,9 @@ export default class Directors extends Mixins(DateMixin, ExternalMixin, EntityFi
   private activeDirectorToDelete = null
   private cessationDateTemp = null
   private isEditingDirector = false
+  private isCompliant = true
```

Comment:
It looks like this isn't used for anything...?

Reply:
that's weird...I thought I removed this, along with the the logging and some other obsolete code referenced in comments...I shall look into this!